setup.py

```diff
@@ -18,24 +18,42 @@
 from pip import req
 import setuptools
 
-
-all_reqs = req.parse_requirements('requirements.txt', session=False)
-test_reqs = req.parse_requirements('test-requirements.txt', session=False)
-version = open(os.path.join(os.path.dirname(__file__),
+VERSION = open(os.path.join(os.path.dirname(__file__),
```

Comment:
Let's not have dependency on 'version.txt' file here, instead let's use `setup.cfg` with `version` attribute defined